### PR TITLE
Fix spelling

### PR DESCRIPTION
**Pull request by JuKu**
Fix spelling

### CODE REVIEW

File: web/templates/user/_user_show_traffic.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{#
```
Comment (Georgeto) [inline web/templates/user/_user_show_traffic.html]: Why is there a change here? Did you add/remove a byte-order mark?

Comment (ibot3) [reply web/templates/user/_user_show_traffic.html]: @JuKu 
There is still a change there for no reason.

**Comment by JuKu** [comment]
@sebschrader @Georgeto Is there any progress here?
Or can we close / delete this PR?

**Comment by lukasjuhrich** [comment]
@JuKu as has been mentioned [here](https://github.com/agdsn/pycroft/pull/266/files#r243905357), it still contains independent changes. please amend that, and force-push your `develop` branch.

**Pull request closed without merging**